docs/requirements.txt

```diff
@@ -1,2 +0,0 @@
-myst-parser
-IPython  # For debugging
```

Comment:
I figure the docs should test the jupyterhub_sphinx_theme's dependencies directly, but as installing a package in the parent folder is a bit messy I figured this is removed and that we let the RTD config / noxfile config install the package itself directly without referencing this file.